curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-fetch-promises-and-async-await-by-building-an-fcc-forum-leaderboard/645f7879ebbdb201892e55e1.md

```diff
@@ -0,0 +1,306 @@
+---
+id: 645f7879ebbdb201892e55e1
+title: Step 106
+challengeType: 0
+dashedName: step-106
+---
+
+# --description--
+
+In this step, you will construct the `avatarUrl` by checking if the `avatar` string starts with `/user_avatar/`. If it does, you will concatenate it with the `AVATAR_URL`. If it doesn't, you will simply use the `avatar` as it is. This will ensure the avatar URL is correctly formed whether it's a relative or absolute URL. You can use a ternary operator to accomplish this.
```

Comment:
Maybe instead of literal concatenation (`AVATAR_URL.concat(...)`), expect both to be joined in the template literal?
```
`${AVATAR_URL}${avatar}`
```